build.go

```diff
@@ -47,6 +47,7 @@ func (cmd *buildCommand) Register(fs *flag.FlagSet) {
 	fs.Var(&cmd.buildArgs, "build-arg", "Set build-time variables")
 	fs.Var(&cmd.labels, "label", "Set metadata for an image")
 	fs.BoolVar(&cmd.noConsole, "no-console", false, "Use non-console progress UI")
+	fs.BoolVar(&cmd.noCache, "no-cache", false, "Do not use cache when building the image")
```

Comment:
line is 88 characters (lll)

Reply:
Can we fix that in a separate PR?
Other lines seem already exceeding the limit.

Reply:
oh sorry this is a stupid bot